ext/LuxReactantExt/training.jl

```diff
@@ -16,10 +16,9 @@ function compute_gradients_internal(objective_function::F, model, data, ps, st)
     return dps, loss, stats, stₙ
 end
 
-function Lux.Training.compute_gradients_impl(
+Profiler.@annotate "Compile Compute Gradients" function Lux.Training.compute_gradients_impl(
```

Comment:
cc @jumerckx we should likely consider adding this to all function calls [and also ops.jl]